### DISCUSSION TITLE
Server detection

### DISCUSSION THREAD

**Discussion by heyhippari** [jellyfin/jellyfin-vue#139] [Features]
If the web client is running on a machine that has a Jellyfin server running (`http://localhost:8096`), the client should use this server by default.

A way to override this should be provided (For example via a JSON config file in the root directory of the built client)

**Comment by phyzical** [comment]
Hey ill take this one, 
Did we want to prefill the server address, but allow the user to change or make it some sort of button to allow the address to be edited?

**Reply by Pfuenzle** [reply]
From what I've seen, this has not been implemented yet. I think the best way would be to skip the server selection and automatically set it to the local server (or one specified in a config file or by the admin), and then have a button next to the login button that says "Change Server" or something similar. I think this would be the most straight forward solution

**Reply by Pfuenzle** [reply]
Hi, sorry to bother you, but has there been any news on this? Would be great to have this implemented

**Reply by phyzical** [reply]
@Pfuenzle sorry i thought this was already solved/ taken off me yonks ago. ill give the comms a review and see if i can have a crack this week

**Reply by Pfuenzle** [reply]
That's great to hear, thanks a lot :)
Looking forward to it
